charts/mattermost-rtcd/templates/daemonset.yaml

```diff
@@ -81,3 +84,6 @@ spec:
       dnsConfig:
         {{- toYaml . | nindent 8 }}
       {{- end }}
+      {{- if .Values.daemonset.extraVolumes }}
+      volumes: {{ toYaml .Values.daemonset.extraVolumes | nindent 8 }}
```

Comment:
I added a NL for readability here, but it messed up the generated YAML. 
Putting these 2 on 1 line seems to fix this.

Reply:
Let's also correctly indent `volumeMounts`:, to align with the rest of the template 


```
          volumeMounts:
          - mountPath: /host/var/log
            name: varlog
            readOnly: true
```

to 
```
          volumeMounts:
            - mountPath: /host/var/log
              name: varlog
              readOnly: true
```

Reply:
Done!